test/templates/blueprint-cli-shared/generators/bar/index.js

```diff
@@ -0,0 +1,7 @@
+const Generator = require('yeoman-generator');
+
+module.exports = class extends Generator {
+    initializing() {
```

Comment:
does it actually do something by default? is that why you've overridden it?

Reply:
generators cannot be empty.